dotcom-rendering/src/model/enhance-H2s.ts

```diff
@@ -63,9 +63,8 @@ const generateId = (element: SubheadingBlockElement, existingIds: string[]) => {
  */
 const enhance = (elements: FEElement[]): FEElement[] => {
 	const slugifiedIds: string[] = [];
```

Comment:
Not your PR but somewhat related: looks like `slugifiedIds` is being mutated as a side effect of `generateId()`

Reply:
Absolutely, this needs a reducer or a for…of loop, I think.

Reply:
Alternative implementation in AR in case it proves useful:

https://github.com/guardian/dotcom-rendering/blob/6c4c762529eeaf813f7409da82091da6b1cb4626/apps-rendering/src/bodyElement.ts#L234-L258